vendor/VRControls.js

```diff
@@ -35,9 +35,9 @@ THREE.VRControls = function ( object, onError ) {
 
 	}
 
-	if ( navigator.getVRDisplays ) {
+	if ( window.navigator.getVRDisplays ) {
```

Comment:
Is it needed in VRControls? This should only be called at render time not start time